ecosystem/platform/server/app/views/onboarding/email.html.erb

```diff
@@ -6,20 +6,28 @@
   <% if current_user.errors.any? %>
     <div id="error_explanation" class="flex p-4 mb-4 bg-red-100 rounded-lg dark:bg-red-200" role="alert">
       <div class="ml-3 text-sm font-medium text-red-700 dark:text-red-800">
-      <h2><%= pluralize(current_user.errors.count, 'error') %> prohibited this user from being saved:</h2>
+        <h2><%= pluralize(current_user.errors.count, 'error') %> prohibited this user from being saved:</h2>
 
-      <ul>
-        <% current_user.errors.each do |error| %>
-          <li><%= error.full_message %></li>
-        <% end %>
-      </ul>
+        <ul>
+          <% current_user.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
       </div>
     </div>
   <% end %>
 
   <%= form_with(model: current_user, url: onboarding_email_path, method: :post, data: { turbo: !@show_recaptcha_v2, controller: 'recaptcha', action: 'recaptcha#validate' }, builder: AptosFormBuilder) do |f| %>
-    <div class="mb-8">
-      <%= f.email_field :email, autofocus: true, autocomplete: 'email', required: true, placeholder: 'ENTER YOUR EMAIL ADDRESS', value: current_user.unconfirmed_email || current_user.email || @oauth_email, class: 'md:w-96' %>
+    <% if !current_user.email %>
```

Comment:
I think this is not email OR not verified, right? so they can go back and re-change this field? (or does that now update the unconfirmed_email field?)

Reply:
The email field isn't populated until the user confirms it (at which point the unconfirmed_email is moved to the email field).